.github/workflows/code-engine-cleanup.yml

```diff
@@ -41,7 +41,7 @@ jobs:
 
       - name: GH deployment status (deactivate)
         id: gh_deployment
-        uses: bobheadxi/deployments@v0.6.2
+        uses: bobheadxi/deployments@v1
         with:
           step: deactivate-env
```

Comment:
now that it has added a `step: delete-env`. i think we should change this to use `delete-env` instead of `deactivate-env`. that way it we dont have an ever growing list of pages and pages of inactive environment hanging around for no reason:
- https://github.com/Qiskit/saiba/deployments/activity_log
- https://github.com/Qiskit/platypus/deployments/activity_log
- https://github.com/Qiskit/qiskit.org/deployments/activity_log

Reply:
Done!